blazor/autocomplete/selection.md

```diff
@@ -0,0 +1,167 @@
+
+N> It will be dependent on the [SortOrder](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SortOrder.html) type. It will bind the value to the component with the sorted data if the corresponding property is defined.
+
+The following sample shows how to bind the index on the initial rendering.
+
+{% highlight cshtml %}
+
+{% include_relative code-snippet/selection/preselect-value-index.razor %}
+
+{% endhighlight %}
+
+![Blazor AutoComplete with bind-index](./images/selection/blazor_autocomplete_preselect-value-index.png)
+
+## Autofill the selected value
+
+Specifies whether the input field of the control will automatically suggest and fill in the first matched item as the user types, based on the items in the control's data source. If no matches are found, the input field will not be filled, and no action will occur. The default value of [Autofill](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.AutoCompleteModel.html#Syncfusion_Blazor_DropDowns_AutoCompleteModel_Autofill) is `false`.
+
+{% highlight Razor %}
+
+{% include_relative code-snippet/selection/auto-fill.razor %}
+
+{% endhighlight %} 
+
+![Blazor AutoComplete with Autofill property](./images/selection/blazor_autocomplete_with-autofill-property.gif)
+
+## Get selected item by value
+
+Get the entire object belonging to the value selected in the component using the [GetDataByValue](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownList-2.html#Syncfusion_Blazor_DropDowns_SfDropDownList_2_GetDataByValue__0_) method.
+
+{% highlight cshtml %}
+
+{% include_relative code-snippet/selection/get-selected-item-by-value.razor %}
+
+{% endhighlight %}
+
+## Focus the next component on selection
+
+Focus the component programmatically using the [FocusAsync](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownList-2.html#Syncfusion_Blazor_DropDowns_SfDropDownList_2_FocusAsync) public method. It will set focus instantly to the AutoComplete component when invoking it. 
+
+{% highlight cshtml %}
+
+{% include_relative code-snippet/selection/focus-next-component.razor %}
+
+{% endhighlight %}
+
+## Prevent reload on form submit
+
+## Programmatically clear the selected value
+
+To clear the Autocomplete value programmatically, use the [ClearAsync](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownList-2.html#Syncfusion_Blazor_DropDowns_SfDropDownList_2_ClearAsync) method. This method clears out the selected values from the SfAutocomplete<TValue, TItem> component and sets the [Value](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownList-2.html#Syncfusion_Blazor_DropDowns_SfDropDownList_2_Value) and [Index](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownList-2.html#Syncfusion_Blazor_DropDowns_SfDropDownList_2_Index) properties to null.
```

Comment:
API links not properly link. Please use autocomplete value and index api link instead of dropdown list.

Reply:
The mentioned properties are not available in the AutoComplete API. I have confirmed this with Saravanan, sir